proto/rill/runtime/v1/expressions.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
```

Comment:
Question: not sure what I think, but do you think we should consider shorter type names given the nesting of expressions and how often we'll end up looking at them printed?

I.e. using terms like `expr` (expression), `cond` (condition), `val` (value), `op` (operation), `args` (operands), `ident` (identifer), `eq` (equal), etc.?

For example, MongoDB does that: https://www.mongodb.com/docs/manual/reference/operator/query/expr/.

Reply:
I would also suggest looking out for prior art for expressions expressed in protobufs. Two good places to look are:
- https://github.com/googleapis/googleapis
- https://buf.build/explore